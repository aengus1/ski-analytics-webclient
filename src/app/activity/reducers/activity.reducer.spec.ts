import { reducer, initialState } from './activities.reducer';

describe('Activity Reducer', () => {
  describe('unknown action', () => {
    it('should return the initial state', () => {
      const action = {} as any;

      const result = reducer(initialState, action);

      expect(result).toBe(initialState);
    });
  });
});
