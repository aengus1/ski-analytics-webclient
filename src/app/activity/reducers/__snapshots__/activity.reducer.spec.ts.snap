// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activity Reducer should return the default state 1`] = `
Object {
  "activitySport": Array [
    "GENERIC",
    "RUNNING",
    "CYCLING",
    "TRANSITION",
    "FITNESS_EQUIPMENT",
    "SWIMMING",
    "BASKETBALL",
    "SOCCER",
    "TENNIS",
    "AMERICAN_FOOTBALL",
    "TRAINING",
    "WALKING",
    "CROSS_COUNTRY_SKIING",
    "ALPINE_SKIING",
    "SNOWBOARDING",
    "ROWING",
    "MOUNTAINEERING",
    "HIKING",
    "MULTISPORT",
    "PADDLING",
    "ALL",
    "INVALID",
  ],
  "activitySubSport": Array [
    "GENERIC_SUBSPORT",
    "TREADMILL",
    "STREET",
    "TRAIL",
    "TRACK",
    "SPIN",
    "INDOOR_CYCLING",
    "ROAD",
    "MOUNTAIN",
    "DOWNHILL",
    "RECUMBENT",
    "CYCLOCROSS",
    "HAND_CYCLING",
    "TRACK_CYCLING",
    "INDOOR_ROWING",
    "ELLIPTICAL",
    "STAIR_CLIMBING",
    "LAP_SWIMMING",
    "OPEN_WATER",
    "FLEXIBILITY_TRAINING",
    "STRENGTH_TRAINING",
    "WARM_UP",
    "MATCH",
    "EXERCISE",
    "CHALLENGE",
    "INDOOR_SKIING",
    "CARDIO_TRAINING",
    "INDOOR_WALKING",
    "CLASSIC",
    "SKATE",
    "ROLLER",
    "ALL_SUBSPORT",
    "INVALID_SUBSPORT",
  ],
  "entities": Object {},
  "filteredIndices": Array [],
  "ids": Array [],
  "selectedActivityId": null,
  "sidebarContent": 0,
  "tsLookup": null,
  "unfilteredActivity": Object {
    "array": Array [
      undefined,
      undefined,
      undefined,
      undefined,
      Array [],
    ],
    "arrayIndexOffset_": -1,
    "convertedFloatingPointFields_": Object {},
    "messageId_": undefined,
    "pivot_": 1.7976931348623157e+308,
    "wrappers_": null,
  },
}
`;
